cypress/e2e/tables/smartTable.cy.js

```diff
@@ -0,0 +1,94 @@
+import { SmartTable } from "../../pageObjects/tables/SmartTable";
+
+describe("Table of users", () => {
+  it("should create and edit user", () => {
+    const user = new SmartTable();
+
+    const userData = {
+      id: 8,
+      firstName: "Yuliia",
+      lastName: "Kovalchuk",
+      username: "@juju",
+      email: "yuliia@gmail.com",
+      age: 31,
+    };
+
+    const updatedUserData = {
+      id: 746,
+      firstName: "Julie",
+      lastName: "Kov",
+      username: "@omgjulie",
+      email: "omgjulie@gmail.com",
+      age: 25,
+    };
+
+    user.navigate();
+    user.createNewUser(
+      userData.id,
+      userData.firstName,
+      userData.lastName,
+      userData.username,
+      userData.email,
+      userData.age,
+    );
+
+    cy.get("table tbody tr:first-child td:nth-child(2)").should(
```

Comment:
you can also move checks to page object method

Reply:
created a new method **checkDataValidation**